### ISSUE TITLE
Clean up license data issues

### ISSUE THREAD

**Issue by anjackson** [ukwa/w3act#364]
Following on from #334, the permission system should be working going forward, but we have some messy data to clean up.

- Mis-inherited licenses, e.g. the twitter account problems, where the old system added an explicit license and licenseStatus to multiple Targets that had no crawlPermission.
- Lost crawlPermissions due to the old code that allowed them to be discarded when a Target was updated.
- Imported licences - the Selective Archive licensed material was imported, setting only the License but not the licenseStatus, and without importing the actual permissions data.

**Comment by GilHoggarth** [comment]
@anjackson What's the sit. with this ticket - does it need to be completed before release of v1.0.6 or can it be postponed? If need, who can do the work? Nils, Roger?

**Comment by PsypherPunk** [comment]
Re. the last bullet point, as per #354, the update functionality is now working. When doing the migration I'll track Targets which already exist in w3act so we can add existing license information.

**Comment by anjackson** [comment]
The initial clean up is done, and further clean up will wait until after we've been able to import WCT/SPT data.
